api-src/org/labkey/api/targetedms/ITargetedMSRun.java

```diff
@@ -26,13 +26,34 @@
  */
 public interface ITargetedMSRun
 {
-    public Container getContainer();
-    public String getBaseName();
-    public String getFileName();
-    public String getDescription();
-    public Date getCreated();
-    public long getId();
-    public Integer getDataId();
-    public Integer getSkydDataId();
-    public String getSoftwareVersion();
+    /** Don't change the ordering of these enum values without updating the values in targetedms.runs.representativedatastate */
+    enum RepresentativeDataState
```

Comment:
I know this predates the move, but I think we should rename one of the two enums named RepresentativeDataState. Maybe this one should be RunRepresentativeDataState?